rpcudp/protocol.py

```diff
@@ -26,6 +26,9 @@ def connection_made(self, transport):
 
     def datagram_received(self, datagram, address):
         log.debug("received datagram from %s", address)
+        asyncio.ensure_future(self.solve_datagram(datagram, address))
+
+    async def solve_datagram(self, datagram, address):
```

Comment:
A few small things:
1. The rest of the code uses the decorator `@asyncio.coroutine`, for consistency that should be used instead of `async def`.
2. All private methods have a `_` prefix - for consistency this method should as well.